params_templ_yyp.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 0.716
+    
+# e.g. "anas_krystallinus"    
+species.name <- Yanyi_Pu
```

Comment:
I think the parameter needs double quotes around it here.

```suggestion
species.name <- "Yanyi_Pu"
```